src/Card.js

```diff
@@ -0,0 +1,21 @@
+import { useState } from 'react';
+import './Card.css'
+
+
+const Card=({img,cardTitle})=> {
+
+    return (
+      <div className="cards">
+        <div class="photo">
+          <a href="#">
+          <img class="logo" src={img}></img>
+          </a>
+        </div>
+        <div class="Title">
+           <h4>{cardTitle}</h4>
+        </div>
+        
+      </div>
+    );
+  }
+export default Card;
```

Comment:
Please add EOL. Also, I think Prettier is not enabled on save. Please check that.